x-pack/plugins/fleet/server/services/artifacts/mocks.ts

```diff
@@ -77,10 +82,12 @@ export const generateEsRequestErrorApiResponseMock = (
   );
 };
 
-export const generateArtifactEsGetSingleHitMock = (): SearchHit<ArtifactElasticsearchProperties> => {
+export const generateArtifactEsGetSingleHitMock = (
```

Comment:
FYI: Just making this mock provider a bit more robust by accepting on input an optional artifact